app/controllers/facilities_controller.rb

```diff
@@ -1,4 +1,13 @@
 class FacilitiesController < ApplicationController
+
+  def index
+    @facilities = Facility.all
```

Comment:
I'm assuming this will later be only the facilities that user has the right to view. Yes?